src/basilisp/lang/runtime.py

```diff
@@ -197,8 +198,6 @@ class RuntimeException(Exception):
 
 
 class _VarBindings(threading.local):
-    __slots__ = ("bindings",)
```

Comment:
I had missed from the [`_threading_local.py` docstring](https://github.com/python/cpython/blob/2085bd0877e17ad4d98a4586d5eabb6faecbb190/Lib/_threading_local.py#L107-L115) that `__slots__` in subclasses of `threading.local` are not themselves thread-local.